### PR TITLE
Rename: Fhi.VersionApiClient.sln -> Fhi.VersionApiClient.sln

### PR DESCRIPTION
**Pull request by GeirMG**
Merge before: https://github.com/FHIDev/Fhi.VersionApiClient/pull/9

### CODE REVIEW

**Comment by GeirMG** [comment]
@OsirisTerje Should we rename the GitHub repo also, and update the README.md?

**Comment by OsirisTerje** [comment]
> should we rename the GitHub repo also, and update the README.md?

sure!

File: Fhi.Common.VersionApiClient.sln
Comment (OsirisTerje) [inline Fhi.Common.VersionApiClient.sln:1]: Navnet skal da være Fhi.VersionApiClient

**Review by OsirisTerje — CHANGES_REQUESTED** [review]

**Comment by GeirMG** [comment]
Replaced by: https://github.com/FHIDev/Fhi.VersionApiClient/pull/11

**Pull request closed without merging**